setup-test-env.js

```diff
@@ -0,0 +1,5 @@
+import { createSerializer } from "jest-emotion"
+import * as emotion from "@emotion/core"
```

Comment:
I thought when we chatted on Thursday we had decided not to use emotion, correct? would it be possible to remove it?

Reply:
Hi I realized that was how I was able to make snapshots, so I made the decision to use it.